src/uct/sm/mm/mm_ep.c

```diff
@@ -39,7 +47,7 @@ static UCS_CLASS_CLEANUP_FUNC(uct_mm_ep_t)
     ucs_status_t status;
 
     /* detach the remote proceess's shared memory segment (remote recv FIFO) */
-    status = uct_mm_pd_mapper_ops(iface->super.pd)->release(self->remote_mem);
+    status = uct_mm_pd_mapper_ops(iface->super.pd)->detach(&self->mapped_desc);
```

Comment:
let's make attach and detach consistent - either both get the struct, or both get address and cookie as individual parameters